src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -530,17 +530,21 @@ internal static object[] GetCustomAttributes<T>(Assembly assembly)
         #region Encoding
 
         /// <summary>
-        /// Facade for Encoding.Default
+        /// Facade for getting Encoding.Default
```

Comment:
Depending on what we will do when `Encoding.Default` is back in netstandard2.0. If we are not going to change to `Encoding.Default` in powershell core, then the comment should be `Facade for getting default encoding`.